resources/views/registration/participantStored.blade.php

```diff
@@ -11,11 +11,15 @@
 <hr/>
 
 <p>
-	U heeft uw kind succesvol ingeschreven voor een Anderwijskamp. U ontvangt een automatische bevestigingsmail op het opgegeven emailadres. Om de inschrijving definitief te maken, dient u het kampgeld over te maken op onze rekening. 
-	@if ($toPay == 0)
-		<strong>Voor dit kamp is de prijs echter nog niet definitief vastgesteld.</strong> Zodra dat is gebeurd, ontvangt u daarover per email bericht.
-	@else
-		De betalingsinformatie vindt u in de bevestigingsmail. Het is handig om dit zo snel mogelijk te doen, want plaatsing voor een kamp gebeurt op volgorde van betaling.
+	U heeft uw kind succesvol ingeschreven voor een Anderwijskamp.
+
+	@if (!$payment->isFree())
+		U ontvangt een automatische bevestigingsmail op het opgegeven emailadres. Om de inschrijving definitief te maken, dient u het kampgeld over te maken op onze rekening.
```

Comment:
De bevestigingsmail krijg je ook als het kamp gratis is, dus die zin moet boven de `@if`